modules/javascript/src/main/webapp/javascript/atmosphere.js

```diff
@@ -347,14 +364,6 @@
             /** Key for connection sharing */
             var _sharingKey;
 
-            /**
-             * {boolean} If window beforeUnload event has been called.
-             * Flag will be reset after 5000 ms
-             *
-             * @private
-             */
-            var _beforeUnloadState = false;
```

Comment:
This was never accessed. Rather, the nonexistent `atmosphere._beforeUnloadState` was. I moved this declaration to line 52.